### PR TITLE
feat: add component for header section style and get data

### PR DESCRIPTION
**Pull request by zeelrupapara**

### CODE REVIEW

File: pages/jeel/index.vue [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    <Middledata :ip-data="ipData" />
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'IndexPage',
+  data() {
+    return {
+      ipAddress: '',
```
Comment (codesnail21) [inline pages/jeel/index.vue:17]: if you added this PR for head section and input data then why is this all map data fields are there ???

**Review by codesnail21 — CHANGES_REQUESTED** [review]
remove the extra thing

File: pages/jeel/index.vue (hunk 1 above)
Comment (zeelrupapara) [reply pages/jeel/index.vue:17]: i just metion which value inside ipData

**Review by Payal17122000 — COMMENTED** [review]
remove extra code

**Pull request merged**